### PR TITLE
Fix ignoring files in folders while syncing

### PR DESCRIPTION
**Pull request by playertwo**
it fix "Sync local -> remote" function. I'm not a javascript developer, so excuse for new bugs :)

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -360,33 +359,25 @@ export default class Client {
...
-
-    return false;
+    this.ignoreFilter = ignore().add(ignoreFile.readSync(true));
+    return true;
   }
 
-  checkIgnore(local) {
-    let haseIgnore = true;
+  checkIgnore(filepath) {
+    filepath = this.toRelative(filepath);
```
Comment (icetee) [inline lib/client.js]: Please no redefine the parameter. Create new `const`.

Comment (playertwo) [reply lib/client.js]: fix it!

Comment (icetee) [reply lib/client.js]: I found another mistake. I'll send the repair right away.

(If you start with "/", subfolder filtering will not work)

**Comment by icetee** [comment]
@playertwo Please test it. I added two commit. Don't forget to install the new packages. `npm install`

**Comment by playertwo** [comment]
@icetee Everything works fine

**Pull request merged**